src/CommandingOfficers/Commander.java

```diff
@@ -95,12 +96,22 @@ public Commander(CommanderInfo info, GameScenario.GameRules rules)
     myAbilities = new ArrayList<CommanderAbility>();
   }
 
+  @Override
+  public void registerForEvents(GameInstance game)
```

Comment:
I'd probably support changing this to `initForGame` as well since it's not just registering for events anymore.

Reply:
It's from the `GameEventListener` interface; `CommanderAbility` isn't (necessarily) a listener, while `Commander` always is.
That said, the registration is always handled separately anyway, so sure.